### DISCUSSION TITLE
chia wallet show json output

### DISCUSSION THREAD

**Discussion by happycouak** [Chia-Network/chia-blockchain#7625] [Ideas]
In order to ease transfers automation of pool rewards to cold wallet, it would be nice to have a json output of "chia wallet show".

**Comment by RudolfAchter** [comment]
If you still looking for this ->

[https://github.com/Chia-Network/chia-blockchain/wiki/RPC-Interfaces/85e31cab243436c5687526507805f9aa2dbe98c3#get-wallet-balance](https://github.com/Chia-Network/chia-blockchain/wiki/RPC-Interfaces/85e31cab243436c5687526507805f9aa2dbe98c3#get-wallet-balance)
